src/server/qgsserverplugins.cpp

```diff
@@ -73,7 +73,10 @@ bool QgsServerPlugins::initPlugins( QgsServerInterface *interface )
 
   QgsDebugMsg( QStringLiteral( "Python support library's instance() symbol resolved." ) );
   sPythonUtils = pythonlib_inst();
-  sPythonUtils->initServerPython( interface );
+  if ( sPythonUtils )
+  {
+    sPythonUtils->initServerPython( interface );
```

Comment:
@nyalldawson , could you avoid the second sPythonUtils null check below by moving the sPythonUtils->isEnabled() check in this conditional branch?

Reply:
Potentially, but it means the fail path in the following check would need to be duplicated (or handled differently)... ultimately it's about the same amount of code